.air.conf

```diff
@@ -0,0 +1,49 @@
+# Config file for [Air](https://github.com/cosmtrek/air) in TOML format
```

Comment:
Adding this in case anyone wants to use it. Not going to add it to the main dev setup though, unless folks really want it.

Reply:
Can we add this to the readme somewhere?